src/main/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/controller/VisitSessionTemplateController.kt

```diff
@@ -118,33 +72,4 @@ class VisitSessionTemplateController(
     @Schema(description = "Template id", example = "45645", required = true)
     @PathVariable templateId: Long
   ): SessionTemplateDto = sessionTemplateService.getSessionTemplates(templateId)
-
```

Comment:
@tpmcgowan Can we remove the delete end point also?

Reply:
@AledWEvans what if we do not remove the endpoint instead throw an Exception every time it's called as these endpoints will be needed in a while?

Reply:
Well thats kind of the hole point is to remove it,... it's code that is based on NO requirements, if we want to revert it at a later date we can go back in git history and restore it! The reason why I want it removed is it's muddies the water and can involve refactoring it, even when we don't use it. It's actually simple code also so would just take us no time at all to re-write it.

Reply:
> @tpmcgowan Can we remove the delete end point also? could not spot it in the UI code!

Yeah, don't see why not. We don't do anything with session templates in the front-end; we only query the available slots that are derived from them.

We've used these endpoints occasionally to create/delete specific template scenarios for testing but we can do that directly in the DB locally, for example, when needed.